ssl_customcerts/all.yml

```diff
@@ -0,0 +1,59 @@
+- hosts: preflight
+  tasks:
+  - import_role:
+      name: confluent.preflight
+- hosts: ssl_CA
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.ssl_customcerts
+- hosts: zookeeper
+  tasks:
+  - import_role:
+      name: confluent.zookeeper
+- hosts: broker
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.kafka-broker
+- hosts: schema-registry
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.schema-registry
+- hosts: connect-distributed
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.connect-distributed
+- hosts: ksql
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.ksql
+- hosts: control-center
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.control-center
+- hosts: kafka-rest
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.kafka-rest
+- hosts: tools
+  tasks:
+  - name: Include TLS vars
+    include_vars: ssl_customcerts/vars/security_vars.yml
+  - import_role:
+      name: confluent.tools
```

Comment:
what does tools role do? which hosts should be in the tools group?

Reply:
It's legacy, not sure the original intent.  My thinking is to leave this in place for this release, and review if we still need it for the next one.  Please a file a bug for this.